publication-commons/src/main/java/no/unit/nva/publication/RequestUtil.java

```diff
@@ -93,14 +102,14 @@ private static UserInstance createInternalUserInstance(RequestInfo requestInfo)
         var personCristinId = attempt(requestInfo::getPersonCristinId).toOptional().orElse(null);
         var topLevelOrg = attempt(requestInfo::getTopLevelOrgCristinId).map(Optional::get).toOptional().orElse(null);
         var accessRights = requestInfo.getAccessRights();
-        return new UserInstance(owner, customerId, topLevelOrg, personCristinId, accessRights);
+        return new UserInstance(owner, customerId, topLevelOrg, personCristinId, accessRights, UserClientType.INTERNAL);
     }
 
     public static UserInstance createUserInstanceFromRequest(RequestInfo requestInfo,
                                                              IdentityServiceClient identityServiceClient)
         throws UnauthorizedException {
         try {
-            return requestInfo.clientIsThirdParty()
+            return requestInfo.clientIsThirdParty() || requestInfo.clientIsInternalBackend()
                        ? createExternalUserInstance(requestInfo, identityServiceClient)
                        : createInternalUserInstance(requestInfo);
```

Comment:
Strange naming clientIsInternalBacked -> createExternalUser.. :)

Reply:
I agree, not sure if I want to change requestInfo.clientIsInternalBackend today, but I will change the other functions to differentiate